bullet-featherstone/src/SDFFeatures.cc

```diff
@@ -135,9 +135,11 @@ std::optional<Structure> ValidateModel(const ::sdf::Model &_sdfModel)
   const ::sdf::Joint *rootJoint = nullptr;
   bool fixed = false;
   const std::string &rootModelName = _sdfModel.Name();
+  std::unordered_map<const ::sdf::Link*, std::vector<const ::sdf::Link*>>
+    linkTree;
```

Comment:
A comment describing the contents of `linkTree` would be helpful; I believe it is a map from each parent link to a vector of its child links.

Reply:
added comment. [e689f8b](https://github.com/gazebosim/gz-physics/pull/562/commits/e689f8b9513c7aa3e94eea73b2979f883699cf3d)